diff --git a/platform/Device/component/efr32zg23b010f512im40.slcc b/platform/Device/component/efr32zg23b010f512im40.slcc
index 39e7ed7e2..0dce2b0f6 100644
--- a/platform/Device/component/efr32zg23b010f512im40.slcc
+++ b/platform/Device/component/efr32zg23b010f512im40.slcc
@@ -165,6 +165,7 @@
   - name: device_supports_connect
   - name: device_supports_zwave
   - name: device_supports_zwave_end_device
+  - name: device_supports_thread
   - name: device_compute_basic
 - recommends:
   - id: trustzone_unaware
diff --git a/platform/Device/component/efr32zg23b010f512im48.slcc b/platform/Device/component/efr32zg23b010f512im48.slcc
index b21a3efa4..230470121 100644
--- a/platform/Device/component/efr32zg23b010f512im48.slcc
+++ b/platform/Device/component/efr32zg23b010f512im48.slcc
@@ -167,6 +167,7 @@
   - name: device_supports_connect
   - name: device_supports_zwave
   - name: device_supports_zwave_end_device
+  - name: device_supports_thread
   - name: device_compute_basic
 - recommends:
   - id: trustzone_unaware
diff --git a/platform/Device/component/efr32zg23b011f512im40.slcc b/platform/Device/component/efr32zg23b011f512im40.slcc
index e8925298e..98f3ebe2c 100644
--- a/platform/Device/component/efr32zg23b011f512im40.slcc
+++ b/platform/Device/component/efr32zg23b011f512im40.slcc
@@ -165,6 +165,7 @@
   - name: device_supports_connect
   - name: device_supports_zwave
   - name: device_supports_zwave_gateway
+  - name: device_supports_thread
   - name: device_compute_basic
 - recommends:
   - id: trustzone_unaware
diff --git a/platform/Device/component/efr32zg23b020f512im40.slcc b/platform/Device/component/efr32zg23b020f512im40.slcc
index 4256a79f0..05278551d 100644
--- a/platform/Device/component/efr32zg23b020f512im40.slcc
+++ b/platform/Device/component/efr32zg23b020f512im40.slcc
@@ -165,6 +165,7 @@
   - name: device_supports_connect
   - name: device_supports_zwave
   - name: device_supports_zwave_end_device
+  - name: device_supports_thread
   - name: device_compute_basic
 - recommends:
   - id: trustzone_unaware
diff --git a/platform/Device/component/efr32zg23b020f512im48.slcc b/platform/Device/component/efr32zg23b020f512im48.slcc
index 2724d9e26..3048a29fb 100644
--- a/platform/Device/component/efr32zg23b020f512im48.slcc
+++ b/platform/Device/component/efr32zg23b020f512im48.slcc
@@ -167,6 +167,7 @@
   - name: device_supports_connect
   - name: device_supports_zwave
   - name: device_supports_zwave_end_device
+  - name: device_supports_thread
   - name: device_compute_basic
 - recommends:
   - id: trustzone_unaware
diff --git a/platform/Device/component/efr32zg23b021f512im40.slcc b/platform/Device/component/efr32zg23b021f512im40.slcc
index 80d63d09b..e8719df6b 100644
--- a/platform/Device/component/efr32zg23b021f512im40.slcc
+++ b/platform/Device/component/efr32zg23b021f512im40.slcc
@@ -165,6 +165,7 @@
   - name: device_supports_connect
   - name: device_supports_zwave
   - name: device_supports_zwave_gateway
+  - name: device_supports_thread
   - name: device_compute_basic
 - recommends:
   - id: trustzone_unaware
